packages/manager/apps/telecom/src/components/notification/notification-list.html

```diff
@@ -93,6 +113,14 @@ <h2 data-translate="components_notification_loading"></h2>
                     data-ng-if="!element.editMode"
                     data-title="{{ 'components_notification_type' | translate }}"
                 >
+                    <em
+                        data-ng-if="element.type === 'sms' "
+                        class="fa fa-mobile"
```

Comment:
another option is to use ng-class for applying classes conditionally

Reply:
I made this change. Thank you for your feedback.